.vscode/launch.json

```diff
@@ -1,28 +0,0 @@
-{
```

Comment:
These files help with VSCode, which I use, and which should be checked into the repo and not removed.

Reply:
reverted